profiling/proto/service/service.go

```diff
@@ -0,0 +1,28 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package service is an empty package that exists so as to allow the `make
```

Comment:
You can move the //go:generate to the actual service implementation.  That mirrors what we do elsewhere in the grpc-go tree.

Reply:
I see, I didn't know that, thanks for letting me know. Done.